.travis.yml

```diff
@@ -8,49 +8,26 @@ before_install:
 - command -v meteor >/dev/null 2>&1 || curl https://install.meteor.com | /bin/sh
 before_script:
 - export PATH=$HOME/.meteor:$PATH
-env:
-  global:
-  - secure: KH6jJVUc12ocK5FyWmDMWeBQZKpEXmpmAyLHnMW4SVxeMCsUCnbKf3sC79ChwTPPEyBWZ4b3Y+lAdNDMhvk3U58QmyjYdPbbEyKK384uvrRrJ//fcg+kQcymEi56V1sZ7qq25G0YZUQOPsv/PnE17AW2fDKrD/50vcCU/zn3hjrilVr4Qwb8act0jCJXZo0TXN5P3qwrsAD7g9cENpIRK3VOldmZALeps1VmXnXbG8zcwAJ8AHSh64WfkXdDLXAkQdnYybyVXUFR9k3gicouy3Mf6rR/N+0gGDItifaWneqCYAXPVg7r6lt9fuHGwbTAUyoV1/zPDfN/7CRGQd0LDHG9f0tThFo/ijJtdQTJeohchuB5XwSABr7rKdfvXtuAz/RJ7Hy9BZZBs8e6/RkA6QLj3VEFejAM+ncP+J/aNwAKJyvAFaKTGUZJKrXkQQ1Y5APEQ1S2SPwRTHUxIgoEMl0yflyJ7nrJpcfTDPtxWFG2tONnMn2J70KBo8yQmHpGyd1hPnLZUvdNdHJsYkuXnYQ13vtOT6sZt6fS4kaJRX4Uxf5u3Flq6KhN3BYty2T9ECc6hTyTdnKHmI/3FGsPEfHi/WWO8B/ylukT4y4h6vyvP+GBPeUNXFbRLmnk8dOrU3DqNd1Ta/PZwopLInbtxvyKKWiH7vJB/nad5Al0iO4=
-  - secure: rXw5vCn+dOVJvMAdnvW0j+t1hSY1I5pvk7vyXgLy4b1XKX/dJ8AJCOfeKGTcIlpKPYj4CipQn1g2yqVznBRhRNylt4rdCRSCIgZE0zUXSw9nDJ4OTz9mGW0gSEvHvCntlCcnG3UR8sO4WBJxysycpC37x8XyoGeRj5yEhahNqkh1U5R7ZVDlJKfo+el+FbFe2poOgffCrBahAYEMy5vtTVmnb+vwAejq2/1nS20EwxieVP3maroMalWkQKLkku9s10SpgQvK9FmCr02FWew7NnouzSROZr+t3mL0xv9CHN/6/83tSd+lDJFfRitHTImcQfIbvKoNpnPVqRuLemB7Jo6BWU/NlKcR0/s0Ef1ewT3ONrSrk41/8pRB0Vm24gc1EAL0Uzi9vl4PDQsEf4vxEt5AAdL2aJemVSRWORs5Wvm1qdB6hgYTzedZV/+pXFQrot8Fw1SzlHhb3ASm8/qTJIHNPCXUDHQgbNN1MconZoTDDjeythe2GyfnfCLMq0aMDGp1nes4CA/8miLkubm5iD9VgXiWAVPXxxWifwx59uph12OooFMWpNQHH7MoztflAsMTQ/bbHRVibp0J5FlGYxncOjCNJwgtShpEA2mICGB2IG75cu4asmDprABK7MP+qEe01hpL2dzGuxBY8/ckIwcbsEOQItPiRMmr7KXXz8c=
-  - SENTRY_ORG=lesswrong
-  - secure: aqPakii7j/8zWAgEwGAwgQfYFEYrA6WJeS2iZBJ8vOpzHk5jGuOKDsCKtFvTQ4VqE9MdXyCjnYalEseS9t5VwwFskf/e7NqhVNld0whPQEBqGKuR9PIzXwlvKeQZ2i34wVW6nCh3H+nDWjXLpSluyj7lN23Yog9Lrtpi/bs/dYualw/lseQH7ZbNwOFIdHpbKgZS8jJ+0LAc8lAlT/UldJGUgL72r2S4wtCb+9wMf5uUoA8IZrdn08lm1owmg8bzfxAwWaJSrV2xnOG2mHUC3IPwrzgGD0IOxhwB/hA7wLYBx982zHRCAh3kKd3kjqgIAAdvt1hMm86eiPM4i4ofa+O0O5/kxiBKS9fRxrFSsLxHvg5kbsgjsT0aNiisk9yqy22VCqGa8JilnjqWXiB0YpJIvE99Mba8zNfLqS+P2R+GjOOqo5+1Jgo1HCZonJABY4IuRw0FTI3EE2BOVfjfECJa7eu96udfKN323NVconsWSCejDgoF8kt0W5c6qNF4QSJSHWYQR2kDrfB8JxAeHoZK7GKGJKuIvAMiIp9OZc+UEV5lLy3Cb+f1Lpjq9Fjamvsa/SsawC5Ix4YQfB19oin86mMlJKrA1GBSPYSOKxQxjQ/2gfx9kYTf/OEQJeEtwGVD8v/spAVOKcjaOKKL2hVFrrnZA2p3nC+VkjzbNmE=
 jobs:
   include:
   - stage: everything
     name: "Tests"
     script: yarn run test && yarn run lint && yarn run checktypes
   - script: skip
-    name: "Deploy LessestWrong"
-    if: branch = devel AND type != pull_request
-    deploy: &lessestwrong
+    # TODO: add 'stage' key = 'deploy' so that this step runs in series, not parallel
```

Comment:
Do you want to resolve this TODO?

Reply:
No. If I did before merge I would put TODO; with a semicolon. If you ever see a TODO; in a PR, I've made a mistake.